soccer/gameplay/plays/restarts/basic_direct.py

```diff
@@ -0,0 +1,49 @@
+import robocup
+import constants
+import standard_play
+import enum
+import behavior
+import main
+import evaluation
+import tactics.coordinated_pass
+import skills
+
+
+class DirectFreeKick(standard_play.StandardPlay):
+
+    class State(enum.Enum):
+        kicking = 1
+        passing = 2
+    def __init__ (self):
```

Comment:
Add a newline between the state class and __init__ function please. This would be fixed by make pretty :(